src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -190,7 +190,8 @@
    id       :- schema/Int
    config   :- jruby-schemas/JRubyPuppetConfig
    flush-instance-fn :- IFn
-   profiler :- (schema/maybe PuppetProfiler)]
+   profiler :- (schema/maybe PuppetProfiler)
+   shared-terminus-state :- (schema/maybe java.util.Map)]
```

Comment:
Nitpick - could add a `java.util` import and just refer to `Map` here.  I think we mostly have been trying to refer to the most immediate class name in code.